packages/sirius-web/frontend/sirius-web-application/src/router/Router.tsx

```diff
@@ -33,6 +34,7 @@ export const Router = () => {
       <Route path="/new/project/*" element={<NewProjectView />} ErrorBoundary={ErrorBoundary} />
       <Route path="/upload/project/*" element={<UploadProjectView />} ErrorBoundary={ErrorBoundary} />
       <Route path="/projects" element={<ProjectBrowser />} ErrorBoundary={ErrorBoundary} />
+      <Route path="/libraries" element={<LibraryBrowser />} ErrorBoundary={ErrorBoundary} />
```

Comment:
we will move that after all the projects releated routes or before but not just in the middle